docs/src/content/hardhat-chai-matchers/docs/reference.md

```diff
@@ -128,7 +128,7 @@ If the event has arguments, the `.withArgs` matcher can be added:
 
 ```ts
 await expect(token.transfer(address, 0))
-  .to.be.revertedWithCustomError(token, "InvalidTransferValue")
+  .to.be.emit(token, "Transfer")
```

Comment:
This works, but we usually use `.to.emit`

Reply:
Changed from '.to.be.emit' to '.to.emit'.